spec.md

```diff
@@ -29,6 +29,7 @@ The key words "unspecified", "undefined", and "implementation-defined" are to be
 
 At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more [filesystem layer changeset](layer.md) archives that will be unpacked to make up the final runnable filesystem.
 The image configuration includes information such as application arguments, environments, etc.
+The manifest list is a higher-level manifest which points to specific image manifest(s) for one or more platforms
```

Comment:
This line is from image-manifest.md, which uses "manifests".  I'm fine with that or "manifest(s)", but I think we should use the same form in both places.  So either use "manifests" here, or update image-manifest.md to use "manifest(s)".